internals/secrethub/service_aws_init.go

```diff
@@ -56,19 +56,55 @@ func (cmd *ServiceAWSInitCommand) Run() error {
 		cfg = cfg.WithRegion(cmd.region)
 	}
 
+	if cmd.role == "" {
+		role, err := ui.Ask(cmd.io, "What role would you like to use?")
```

Comment:
Do we want to be short or more descriptive here?

Reply:
Yes

Suggestions are welcome :+1:

Reply:
Is the following correct?
`What IAM role do you want to use to decrypt the service's credential?`
or more in terms interesting for the user:
`What IAM role should have access to the service?`

Reply:
What IAM role should have access to the service? (full ARN or role name)